opus/application/static_media/js/browse.js

```diff
@@ -696,90 +701,20 @@ var o_browse = {
-                    for (let count = 0; count < numToDelete; count++) {
-                        o_browse.deleteCachedObservation(galleryObsElem, tableObsElem, count, viewNamespace);
-                    }
-                }
-            }
-
-            let dataResultCount = viewNamespace.totalObsCount;
-            let firstObsInLastRow = (o_utils.floor((dataResultCount - 1)/galleryBoundingRect.x) *
-                                     galleryBoundingRect.x + 1);
-            let maxSliderVal = (firstObsInLastRow - galleryBoundingRect.x *
-                                (galleryBoundingRect.y - 1));
-
-            // Update obsNum in both infiniteScroll instances.
-            // Store the most top left obsNum in gallery for both infiniteScroll instances
-            // (this will be used to updated slider obsNum).
-            // Store the obsNum at current scrollbar location for both infiniteScroll instances
-            // (this wll be used to updated the table view scrollbar location after initInfiniteScroll
-            //  load is triggered).
-            let tableCurrentObsNum = obsNum;
-            obsNum = (o_utils.floor((obsNum - 1)/galleryBoundingRect.x) *
-                      galleryBoundingRect.x + 1);
-
-            if (maxSliderVal >= obsNum) {
-                // "sliderObsNum" will be the startObs.
-                // "scrollbarObsNum" will be the obsNum at current scrollbar location.
-                // In table view, it's the top obsNum. In gallery view, it's one of obsNums in the top row.
-                $(`${tab} .op-gallery-view`).infiniteScroll({
-                    "sliderObsNum": obsNum,
-                    "scrollbarObsNum": tableCurrentObsNum
-                });
-                $(`${tab} .op-data-table-view`).infiniteScroll({
-                    "sliderObsNum": obsNum,
-                    "scrollbarObsNum": tableCurrentObsNum
-                });
-                opus.prefs[startObsLabel] = obsNum;
-
-                $(`${tab} .op-observation-number`).html(o_utils.addCommas(obsNum));
-                $(`${tab} .op-slider-pointer`).css("width", `${o_utils.addCommas(maxSliderVal).length*0.7}em`);
-
-                // just make the step size the number of the obserations across the page...
-                // if the observations have not yet been rendered, leave the default, it will get changed later
-                if (galleryBoundingRect.x > 0) {
-                    o_browse.gallerySliderStep = galleryBoundingRect.x;
-                }
-                $("#op-observation-slider").slider({
-                    "value": obsNum,
-                    "step": o_browse.gallerySliderStep,
-                    "max": maxSliderVal,
-                });
-            }
-            $(`${tab} .op-slider-nav`).removeClass("op-button-disabled");
-            // update startobs in url when scrolling
-            o_hash.updateHash(true);
+            // When we update the slider, the following steps are run:
```

Comment:
I think renaming the variable "numThumbnails" to numObservations <or similar> would help the code at line 699 be clearer because 'selector' may refer to the number of thumbnails on the gallery page OR the number of observations in the viewable part of the table,